frontend/src/components/GifWidget/GifWidget.styled.ts

```diff
@@ -0,0 +1,89 @@
+import styled from "styled-components";
+
+const GifWidgetStyled = styled.div`
+    position: absolute;
+    right: 3rem;
+    bottom: 7rem;
+
+    width: 30rem;
+    height: 35rem;
+    background-color: #000000bb;
+
+    box-shadow: 5px 10px 25px #000000bb;
+    border-radius: 4px;
+    overflow-y: auto;
+
+    &::-webkit-scrollbar {
+        width: 5px;
+    }
+
+    &::-webkit-scrollbar-thumb {
+        border-radius: 3px;
+        background: ${({ theme }) => theme.primary};
+    }
+`;
+
+const GifWidgetContent = styled.div``;
+
+const GifWidgetSearch = styled.input`
+    all: unset;
+
+    width: 16rem;
+    height: 3rem;
+    padding: 0 1rem;
+    margin: 1rem 1rem;
+
+    color: ${({ theme }) => theme.white};
+    background-color: ${({ theme }) => theme.gray_400};
+    border: 1px solid ${({ theme }) => theme.primary};;
+    border-radius: 0.2rem;
+    box-shadow: 5px 10px 25px #000000bb;
+`;
+
+const GifList = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    gap: 0.8rem;
+    height: auto;
+    justify-content: center;
+`;
+
+const GifListImage = styled.img`
+    width: 30%;
+    height: 8rem;
+    object-fit: cover;
+    border-radius: 4px;
+    cursor: pointer;
+`;
+
+const GifWidgetAtrtibutionSection = styled.div`
+    display: flex;
+    align-items: center;
+    gap: .5rem;
+    bottom: 7rem;
+
+    position: fixed;
+    width: 30rem;
+    padding: 0.7rem 1rem;
+
+    font-size: 13px;
+
+    color: ${({ theme }) => theme.white};
+    background-color: ${({ theme }) => theme.gray_400};
+
+    cursor: default;
+`;
+
+const GiphyAttributionLogo = styled.img`
```

Comment:
if u don't want to expand this u can use only html tag:
https://www.w3schools.com/tags/tag_img.asp

Reply:
If u don't want to add more styles here, u can create it as <img width: ... />
The SC component is unnecessary in that case

Reply:
what are you talking about? there is image width set explicitly within GiphyAttributionLogo.

Reply:
Yes, but i say, if u don't want to create more props here, u can remove this component, and create it as a plain <img> tag.
That's my suggestion

Reply:
Your solution is good